api/action/client.go

```diff
@@ -66,6 +66,18 @@ func (c *Client) Enqueue(arg params.Actions) (params.ActionResults, error) {
 	return results, err
 }
 
+// EnqueueV2 takes a list of Actions and queues them up to be executed by
+// the designated ActionReceiver, returning the ids of the overall operation
+// and each individual task.
+func (c *Client) EnqueueV2(arg params.Actions) (params.EnqueuedActions, error) {
```

Comment:
This will be the method that sticks around after the old legacy one is removed (when the feature flag goes away) - it'll be a pity to have V2 on the end forever. Change it to EnqueueOperation?

Reply:
ok